packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -4,48 +4,69 @@ import { Button } from './buttons/Button';
 import { ScrollOnboardingVersion } from '../lib/featureValues';
 import { cloudinary } from '../lib/image';
 import AnalyticsContext from '../contexts/AnalyticsContext';
-import { AnalyticsEvent, TargetType } from '../lib/analytics';
+import { AnalyticsEvent } from '../lib/analytics';
 
-const GaussianBlur = (): ReactElement => {
-  return (
-    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
-  );
-};
+type ScrollOnboardingVersionMap = Partial<
+  Record<ScrollOnboardingVersion, string>
+>;
 
-const versionToContainerClassName: Record<ScrollOnboardingVersion, string> = {
-  [ScrollOnboardingVersion.V1]: 'h-60',
-  [ScrollOnboardingVersion.V2]: 'h-[36rem]',
+const versionToContainerClassName = {
+  [ScrollOnboardingVersion.V1]: 'h-48 my-5',
+  [ScrollOnboardingVersion.V2]: 'h-80 m-10',
 };
-const versionToButtonClassName: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonClassName: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'w-[16.25rem]',
   [ScrollOnboardingVersion.V2]: 'w-40',
 };
-const versionToButtonText: Record<ScrollOnboardingVersion, string> = {
+const versionToButtonText: ScrollOnboardingVersionMap = {
   [ScrollOnboardingVersion.V1]: 'Customize your feed',
   [ScrollOnboardingVersion.V2]: 'Start',
 };
 
+const versionToGaussianBlurClassName: ScrollOnboardingVersionMap = {
+  [ScrollOnboardingVersion.V1]: 'top-0 right-0 bottom-0 left-0 m-auto',
+  [ScrollOnboardingVersion.V2]: '',
+};
+
+const GaussianBlur = ({
+  version,
+}: {
+  version: ScrollOnboardingVersion;
+}): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        'absolute w-48 h-48 rounded-full blur-2xl opacity-[0.2] bg-theme-color-cabbage',
+        versionToGaussianBlurClassName[version],
+      )}
+    />
+  );
+};
+
 interface ScrollFeedFiltersOnboardingProps {
   version: ScrollOnboardingVersion;
   onInitializeOnboarding: () => void;
 }
-export default function scrollFeedFiltersOnboarding({
+export default function ScrollFeedFiltersOnboarding({
   version,
   onInitializeOnboarding,
 }: ScrollFeedFiltersOnboardingProps): ReactElement {
   const { trackEvent } = useContext(AnalyticsContext);
   useEffect(() => {
     trackEvent({
-      event_name: AnalyticsEvent.Impression,
-      target_type: TargetType.ScrollBlock,
+      event_name: AnalyticsEvent.EligibleScrollBlock,
       target_id: version,
     });
-  }, []);
+  }, [version]);
+
+  if (version === ScrollOnboardingVersion.Control) {
+    return null;
+  }
```

Comment:
We don't need to define control if it's a empty version.
People will simply not get it active.

Reply:
From the analytics events excel sheet it was defined that `control`, `v1` or `v2`  should be sent.

Reply:
@idoshamun Pretty sure we should not fire event if user doesn't even get it right?

Reply:
I think this is due to `control` also being logged with the new instructions for `eligible scroll block` event.

Reply:
Had a chat with @idoshamun about it yesterday (you can look at the description of the event in the sheet).

Reply:
@rebelchris we should fire the event even if the user doesn't get it. This is required for an accurate analysis.

Reply:
Alright, just something we normally don't do so it was out of the ordinary.